cmd/spire-server/cli/entry/delete.go

```diff
@@ -142,10 +143,17 @@ func (c *deleteCommand) prettyPrintDelete(env *commoncli.Env, results ...any) er
 
 	if len(failed) > 0 {
 		env.Printf("\n\n")
+		summaryMsg := fmt.Sprintf("Deleted %d entries successfully, but failed to delete %d entries", len(succeeded), len(failed))
+
+		if len(succeeded) == 0 {
+			summaryMsg = fmt.Sprintf("Failed to delete %d entries", len(failed))
+		}
+
+		env.Printf("%s\n\n", summaryMsg)
```

Comment:
One last thing. I think this would leave us with 2 empty lines after the output which wouldn't look very nice so probably best tonot have them.

```suggestion
		env.Printf("%s", summaryMsg)
```